### PR TITLE
Ees 1282 attempt to fix ui test failures

### PR DESCRIPTION
**Pull request by markhiveit**

### CODE REVIEW

File: tests/robot-tests/tests/general_public/table_tool_absence_in_prus.robot [hunk 1]
```diff
@@ -20,36 +20,37 @@ Select "Pupil absence" publication
     user opens details dropdown    Pupil absence
     user selects radio      Pupil absence in schools in England
     user clicks element    css:#publicationForm-submit
-    user waits until page contains    Choose a subject
+    user waits until element is visible  xpath://h2[text()="Choose a subject"]
```
Comment (ntsim) [inline tests/robot-tests/tests/general_public/table_tool_absence_in_prus.robot:23]: Would change this to `user waits until page contains heading 2`, then change the keyword (and the other heading keywords) to check on visibility.

Do the same for the rest of the xpath selectors.

File: tests/robot-tests/tests/libs/utilities.py [hunk 2]
```diff
@@ -251,10 +251,13 @@ def user_clicks_indicator_checkbox(indicator_label):
         f'//*[@id="filtersForm-indicators"]//label[contains(text(),"{indicator_label}")]/../input').click()
 
 
-def user_clicks_subheaded_indicator_checkbox(subheading_label, indicator_label):
-    sl.page_should_contain_checkbox(f'//*[@id="filtersForm-indicators"]//legend[text()="{subheading_label}"]/..//label[text()="{indicator_label}"]/../input')
-    sl.driver.find_element_by_xpath(
-        f'//*[@id="filtersForm-indicators"]//legend[text()="{subheading_label}"]/..//label[text()="{indicator_label}"]/../input').click()
+def user_selects_subheaded_indicator_checkbox(subheading_label, indicator_label):
```
Comment (ntsim) [inline tests/robot-tests/tests/libs/utilities.py:254]: Couldn't all of these just be in `common.robot`?

The only complicated part is the selector, and even that can be assigned to a variable in Robot Framework.

Also, as this isn't being wrapped with the try/except, it wouldn't be great from a debug standpoint.

File: tests/robot-tests/tests/libs/utilities.py [hunk 3]
```diff
@@ -263,8 +266,9 @@ def user_checks_indicator_checkbox_is_selected(indicator_label):
 
 
 def user_checks_subheaded_indicator_checkbox_is_selected(subheading_label, indicator_label):
-    sl.checkbox_should_be_selected(
-        f'xpath://*[@id="filtersForm-indicators"]//legend[text()="{subheading_label}"]/..//label[text()="{indicator_label}"]/../input')
+    checkbox_selector = f'xpath://*[@id="filtersForm-indicators"]//legend[text()="{subheading_label}"]/..//label[text()="{indicator_label}"]/../input'
```
Comment (ntsim) [inline tests/robot-tests/tests/libs/utilities.py:269]: Couldn't all of these just be in `common.robot`?

The only complicated part is the selector, and even that can be assigned to a variable in Robot Framework.

Also, as this isn't being wrapped with the try/except, it wouldn't be great from a debug standpoint.

Comment (markhiveit) [reply tests/robot-tests/tests/libs/utilities.py:269]: I don't think the try/catch blocks are required if you're calling selenium library functions directly - I'm assuming they handle that themselves.

Comment (markhiveit) [reply tests/robot-tests/tests/libs/utilities.py:269]: ah - I see  there is one webdriver call - I'll wrap it.

**Pull request merged**